src/flint/flintlib/nmod.pxd

```diff
@@ -1,6 +1,6 @@
 from flint._flint cimport mp_limb_t, mp_bitcnt_t
 
-cdef extern from "flint/nmod_vec.h":
+cdef extern from "flint/nmod.h":
```

Comment:
@deinst I needed to change this for clang (OSX) and Flint 3.

I'm not sure why the problem only shows up with clang and Flint 3. Probably it should changed on master as well: see the last commit in this PR. I don't think it's urgent given that a fix is here ready for Flint 3 when it gets released (I guess that is soon).

Reply:
My wife's sister and her husband are visiting for a few days so my work will be sporadic, but  the new `nmod.pxd` will land next week at the latest.  I have been updating the flintlib submodule with all the functions from the documentation, and have found a couple of doc bugs.  This should be done in the next few days

Reply:
No need to rush!

I doubt there will be a release of python-flint before flint 3 so at least the change above will be merged by then from this PR if not another.

Reply:
Actually I'm putting out a release right now (#69) due to (#66).

It's all fine though I think for now.